<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Window</class>
 <widget class="QWidget" name="Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>836</width>
    <height>506</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RP Renamer</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(106, 106, 106);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="3">
    <widget class="QLabel" name="label">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>15</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(136, 136, 136);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Last Source Directory:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLineEdit" name="dir_edit">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(200, 200, 200);
</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="load_files_button">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(216, 216, 216);</string>
     </property>
     <property name="text">
      <string>&amp;Load Files</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(136, 136, 136);
color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>Files to Rename</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="src_files_list">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(191, 191, 191);</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(136, 136, 136);
color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>Renamed Files</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="dst_files_list">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(191, 191, 191);</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QLabel" name="label_4">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>15</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(136, 136, 136);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Filename Prefix:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLineEdit" name="prefix_edit">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(200, 200, 200);</string>
     </property>
     <property name="placeholderText">
      <string>What do you want to rename your files to?</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLabel" name="extension_label">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(136, 136, 136);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>*.jpg</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QPushButton" name="rename_files_button">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(216, 216, 216);</string>
     </property>
     <property name="text">
      <string>&amp;Rename</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="3">
    <widget class="QProgressBar" name="progressBar">
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
gridline-color: rgb(255, 85, 127);</string>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
